resources/definitions/renkforce_rf100.def.json

```diff
@@ -0,0 +1,231 @@
+{
+    "id": "RF100",
+    "version": 2,
+    "name": "renkforce RF100",
```

Comment:
We'd better capitalise the R. I base that on this: https://www.conrad.de/de/renkforce-rf100-3d-drucker-starter-kit-inkl-filament-single-extruder-1507428.html